commands/docker/docker-tag.md

```diff
@@ -0,0 +1,14 @@
+### Provide new tag to docker image
+
+`docker tag {{EXISTING_IMAGE_NAME[:TAG]/ID}} {{NEW_IMAGE_NAME[:TAG]}}}`
+
+- <b>EXISTING_IMAGE_NAME[:TAG]/ID: </b> Name with tag or image id of the image
+- <b>NEW_IMAGE_NAME[:TAG]: </b>New name for the existing image. If `[:TAG]` is not provided, by default `:latest` tag will be added.
+
+#### Example:
+
+`docker tag ubuntu ubuntu:new`
```

Comment:
`docker tag ubuntu ubuntu:new` => `docker tag ubuntu:latest ubuntu:new`

Reply:
Updated in the new commit